xtas/tasks/single.py

```diff
@@ -282,7 +282,7 @@ def sentiwords_tag(doc, output="bag"):
         return counts
 
     elif output == "tokens":
-        return [ngram if polarity == 0 else (ngram, polarity)
```

Comment:
If the output goes to Elasticsearch, you don't want to index the entire document again, just the parts with positive or negative polarity.